app/src/main/res/values/strings.xml

```diff
@@ -556,4 +556,5 @@
     <string name="downloads_storage_ask_summary_kitkat">You will be asked where to save each download.\nChoose SAF if you want to download to an external SD card</string>
     <string name="downloads_storage_use_saf_title">Use SAF</string>
     <string name="downloads_storage_use_saf_summary">The Storage Access Framework allows downloads to an external SD card.\nNote: some devices are not compatible</string>
+    <string name="delete_downloaded_files">Delete downloaded files</string>
```

Comment:
Done, although I'm not sure about the buttons placement, currently it's clear downloads on top, delete files on bottom and cancel between them.

Reply:
Thanks. Please don't use use "Clear download history" instead of "clear finished downloads". This is more precisely.

> I'm not sure about the buttons placement, currently it's clear downloads on top, delete files on bottom and cancel between them.

I think it is ok to cheat here ;) "Delete downloads" should be at the top, "clear download history" in the middle and "cancel" at the bottom. Please misuse the positive/neutral/negative buttons and add a comment above stating this was done by purpose to achieve a good order.

When delete was selected, please add a snackbar allowing the user to "undo" deleting the files similar to the one used when deleting one file.